esrally/driver/runner.py

```diff
@@ -1433,8 +1464,10 @@ def __call__(self, es, params):
                 # we can determine success if and only if the runner returns a dict. Otherwise, we have to assume it was fine.
                 elif isinstance(return_value, dict):
                     if return_value.get("success", True):
+                        self.logger.debug("%s has returned successfully", repr(self.delegate))
                         return return_value
                     else:
+                        self.logger.debug("%s has returned with an error: %s.", repr(self.delegate), return_value)
```

Comment:
I am wondering if it'd be handy to log this more eagerly than only on debug, for troubleshooting purposes and giving a more complete picture of what's going on without having to switch to debug.

Do you think it'd become very chatty?

Reply:
IMHO it could indeed become a bit chatty because it would add a log entry every time the condition is not yet met.